spec/features/ineligible_tslr_claims_spec.rb

```diff
@@ -43,13 +43,29 @@
     expect(page).to have_text("You must be still working as a teacher to be eligible")
   end
 
-  scenario "did not teach at least half their time in an eligible subject" do
+  scenario "does not teach an eligible subject" do
     claim = start_tslr_claim
     choose_qts_year
     choose_school schools(:penistone_grammar_school)
     choose_still_teaching
 
-    expect(page).to have_text(I18n.t("tslr.questions.mostly_teaching_eligible_subjects"))
+    choose I18n.t("tslr.questions.eligible_subjects.not_applicable")
+    click_on "Continue"
+
+    expect(claim.reload.mostly_teaching_eligible_subjects).to eq(false)
+    expect(page).to have_text("You’re not eligible")
+    expect(page).to have_text("You must have spent at least half your time teaching an eligible subject.")
+  end
+
+  scenario "does not teach an eligible subject" do
```

Comment:
These two scenarios have the same name. Is that right?

Reply:
I've tweaked this to be a bit more descriptive